.github/workflows/publish-auxiliary-packages.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish Auxiliary Packages
+on:
+  # NOTE: avoid using the manual execution unless is the only way to fix some issue,
+  # rather retry failed jobs in case of flakes. The manual execution can potentially
+  # cause the tags to point to a different commit that the one used to publish
+  # the packages
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+        startsWith(github.event.head_commit.message, 'chore(release): bump auxiliary package versions')
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Github App Token
+      uses: mongodb-js/devtools-shared/actions/setup-bot-token@main
+      id: app-token
+      with:
+        app-id: ${{ vars.DEVTOOLS_BOT_APP_ID }}
+        private-key: ${{ secrets.DEVTOOLS_BOT_PRIVATE_KEY }}
+    
+    - uses: actions/checkout@v4
+      with:
+        # don't checkout a detatched HEAD
+        ref: ${{ github.head_ref }}
+
+    - name: "Use Node.js 14"
+      uses: actions/setup-node@v3
+      with:
+        node-version: 20.16.0
+
+    - name: Install npm@10.2.4
+      run: npm install -g npm@10.2.4
+
+    - name: Install Dependencies
+      run: |
+        npm run bootstrap-ci
+      shell: bash
+
+    - name: "Publish what is not already in NPM"
+      env:
+        NPM_TOKEN: ${{ secrets.DEVTOOLSBOT_NPM_TOKEN }}
+      run: |
+        echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
```

Comment:
Can/should we write to `~/.npmrc` instead, so we don't need to be careful around not including this file as we do below? (non-blocking)